devices/roku-tv.groovy

```diff
@@ -19,6 +19,7 @@
 preferences {
     input "deviceIp", "text", title: "Device IP", required: true
     input "deviceMac", "text", title: "Device MAC Address", required: true, defaultValue: "UNKNOWN"
+    input "includeInputChildDevices", "boolean", title: "Include Inputs in App List", required: true, defaultValue: false
```

Comment:
I already added the various inputs over the weekend.  I wanted the ability to turn on/off the AV and Channel app, and to customize the htmi inputs to match what is on the TV, so all this is handled in my new driver.  please take a look at it.